src/booking/OfferRequests/OfferRequestsTypes.ts

```diff
@@ -40,7 +40,8 @@ export interface OfferRequestPassenger {
   /**
    * The type of the passenger.
    */
-  type?: 'adult'
+  //type?: 'adult'
+  type?: string;
```

Comment:
I know we haven't been clear about this in the documentation and we could have improved the types to make this clearer, but this field can actually only be `adult`. The idea is that if the passenger is not an adult, the API would expect the `age` to be provided instead. https://duffel.com/docs/api/v1/offer-requests/create-offer-request (The possible values here aren't up to date - we will fix this shortly!)

<img width="667" alt="image" src="https://user-images.githubusercontent.com/6373626/212761261-ea93ed45-1a1c-4fda-8cd3-5baba7941cbd.png">

So if you would like to contribute to this, I would recommend turning this interface to intersect with the union of `{ age: number, type?: never }` and `{ type: 'adult'; age: never }` so that the type documents the real expectation more accurately. Otherwise, we will get to this once we have a chance.